crates/transaction-emitter-lib/src/emitter/local_account_generator.rs

```diff
@@ -64,3 +69,66 @@ impl LocalAccountGenerator for PrivateKeyAccountGenerator {
         Ok(accounts)
     }
 }
+
+pub struct KeylessAccountGenerator {
+    serialized_proofs: Vec<String>,
+    ephemeral_secret_key: Ed25519PrivateKey,
+    expiry_date_secs: u64,
+}
+
```

Comment:
Let's add support for reading a proof file - the file path can be specified as env variable and initialize the `serialized_proofs` ?